launcher/ui/pages/instance/VersionPage.ui

```diff
@@ -204,14 +202,6 @@
     <string>Install the LiteLoader package.</string>
    </property>
   </action>
-  <action name="actionInstall_mods">
-   <property name="text">
-    <string>Install mods</string>
-   </property>
-   <property name="toolTip">
-    <string>Install normal mods.</string>
-   </property>
-  </action>
```

Comment:
I take it these are no longer used?

Reply:
It's just a duplicate action; it takes you straight to the mods page